extra/mediawiki/api/api.factor

```diff
@@ -0,0 +1,216 @@
+! Copyright (C) 2021 Giftpflanze.
+! See http://factorcode.org/license.txt for BSD license.
+USING: arrays accessors assocs calendar combinators
+continuations formatting http http.client io json.reader kernel
+locals make math math.parser namespaces oauth1 prettyprint
+sequences strings system threads ;
+IN: mediawiki.api
+
+TUPLE: oauth-login consumer-token consumer-secret access-token
+access-secret ;
+TUPLE: password-login username password ;
+
+C: <oauth-login> oauth-login
+C: <password-login> password-login
+
+SYMBOLS: basetimestamp endpoint botflag contact cookies
```

Comment:
It's no big deal, but perhaps these could be more readable with something like ``base-timestamp`` or ``bot?``?

Another idea might be to prefix them as we do in a few places, so ``mediawiki-bot?`` and ``mediawiki-contact``, etc?

Just an idea...

Reply:
I think those are good ideas.